File: xssfinder-java-executor/src/test/java/org/xssfinder/xss/XssAttackFactoryTest.java
package org.xssfinder.xss;

import org.junit.Test;

import static org.hamcrest.Matchers.is;
import static org.hamcrest.Matchers.not;
import static org.hamcrest.Matchers.nullValue;
import static org.junit.Assert.assertThat;
import static org.mockito.Mockito.when;

public class XssAttackFactoryTest {
    @Test
    public void createsXssAttack() {
        // given
        XssAttackFactory factory = new XssAttackFactory();

        // when
        XssAttack attack = factory.createXssAttack("1");

        // then
        assertThat(attack, is(not(nullValue())));
    }
}

File: xssfinder-php-executor/test/XssFinder/Xss/XssAttackFactoryTest.php
<?php

namespace XssFinder\Xss;

class XssAttackFactoryTest extends \PHPUnit_Framework_TestCase
{
    public function testCreatesXssAttack() {
        // given
        $factory = new XssAttackFactory();

        // when
        $attack = $factory->createXssAttack("1");

        // then
        assertThat($attack, is(not(nullValue())));
    }
}